culture/remote/README.md

```diff
@@ -0,0 +1,43 @@
+# Working with our Remote Teammates
+
+At Sparkbox, we have the opportunity to work with many different people in many different locations. Whether its' with our co-workers or our clients, sometimes, we all have to work remote.
+
+## Tips for Non-remote folks When Working With Remote Colleagues
+
+Let's be more inclusive of our remote coworkers and clients. Here are some ways how.
+
+- When a conversation becomes more than a quick back-and-forth on [Slack][], spin up a [Zoom][] and encourage all parties to join.
+- Join meetings 2-5 minutes early, to encourage friendly, human banter. Avoid jumping directly into tasking whenever possible.
+- Use a headset/headphone in public spaces. Avoid noisy or high-traffic areas, even if they don't seem loud to you.
+- Let people know if you're not using a headset, or if they are "on blast" in public spaces.
+- Activities with physical paper, whiteboards, or other items should be avoided. Use online tools such as [draw.io][] instead.
+- When one person is remote on a meeting, everyone should be remote. Find individual spaces, such as your desk, so that everyone uses headphones and is on the call separately. This equalizes the speaking floor so that everyone involved has the same opportunity to share and listen. No person has to compete with a large conference room where the video screen is quickly forgotten.
```

Comment:
This is interesting since it's so different from what we usually do (like when a group of people are in the office, we find a meeting room), but the reason is valid and makes sense.

Reply:
More details here around "when"/what type of meetings this is most important for or when you might not do this. 

People will have to scatter if they are physically in the same space, b/c there is awkward sound seepage that I've had happen trying to do this even in the same room as someone else.

This one feels like it could hurt.

Reply:
This is the only tip I have reservations about. I totally get the benefits as far as equalizing the playing field for everyone, but there are definitely benefits lost to those who can meet in the same room as well. For me personally, being on a Zoom meeting for more than 30 minutes becomes exhausting. Just one example of when I'd much rather give folks the options of sitting in a room together rather then joining a Zoom for 2 hours just because I'm remote that day. On the other hand, this tip feels perfect for something like standup meetings.

Reply:
So, we have had issues with this type of thing recently. Lots of side conversations happening in meeting rooms, sometime close to the mic and drowning out the room conversation. This can often make it hard for the people on zoom to join in because the people gathered together don't have the same delay, and it's easier to pick up on visual clues that somebody wants to talk. That being said, we've also had problems with people getting on individual computers, and not being far enough apart and causing painful audio feedback for those on the zoom.

I'm torn on this one, as I would also rather gather the team together in a space and zoom in people who are remote. We tend to do this here for our big team meetings. I think having someone in the room designated as the Zoom Advocate can help to make sure that everyone is getting opportunities to speak is important.

Reply:
As long as people are muting when they are not speaking, then sound seepage can be kept to a minimum. I will say, it is tough to have several different meetings going on in the big room sometimes with the sound seepage issue, but that says more about the space we're in than anything, I think.

Setting a zoom advocate is a good idea, but I have yet to be in a meeting where that gets implemented (99% of the time it does not) and it works well. I would agree that it is ideal to have people in the same room, but when that's not possible, I think it's important to prioritize the people who have the most restrictions. And having us all operate in a way that we're all facing the same restrictions makes that easier on everyone involved.

Reply:
"Lots of side conversations happening in meeting rooms" - is this with the meeting attendees or outside noise? I think side conversations would be an issue no matter what the call setup.

Reply:
Meeting attendees, usually. Though sometimes it's background noise. Side conversations don't tend to happen much if you don't have a bunch of people in the same room.

Reply:
well, they happen, just over Slack :)

Reply:
Sorry to comment late on this one. I understand the reasoning behind this one, but if I'm honest, not being allowed to group up for meetings in the office makes me sad. It feels like I'm losing one of the great things about coming into the Dayton office for work.

I wish there was a way to make the working experience better for remote people without making it worse for those in the office.

Reply:
You're okay, @bryanbraun! I appreciate you sharing your perspective.